tools/reload_sdk_allowlist.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+SHELL_FOLDER="$(cd "$(dirname $0)" || exit;pwd)"
+
+LOG_ERROR() {
+    content=${1}
+    echo -e "\033[31m[ERROR] ${content}\033[0m"
+}
+
+LOG_INFO() {
+    content=${1}
+    echo -e "\033[32m[INFO] ${content}\033[0m"
+}
+
+check_cal_line()
+{
+    line=$1;
+    if [[ $line =~ cal.[0-9]*=[0-9A-Fa-f]{128,128}$ ]]; then
+        echo "true";
+    else
+        echo "false";
+    fi
+}
+
+check_cal_lines()
+{
+    # print Illegal line
+    config_file=$1
+    error="false"
+    for line in "$(grep -v "^[ ]*[;]" $config_file | grep "public_key\.")"; do
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5822413)